### PR TITLE
Add Pinterest country specific TLDs to social list

### PR DESCRIPTION
**Pull request by boris-hocde**
Pinterest is accessible through different domains, with country codes.
Demo: https://drive.google.com/open?id=18LabOa-iAnVcCM4q_46oduU8BKlrZlgK

### CODE REVIEW

File: Socials.yml [hunk 1]
```diff
@@ -153,7 +153,19 @@ Ozone:
   - qzone.qq.com
 
 Pinterest:
+  - pinterest.ca
+  - pinterest.ch
   - pinterest.com
```
Comment (sgiehl) [inline Socials.yml]: would you mind changing it to have `pinterest.com` the first in the list?

**Comment by boris-hocde** [comment]
@sgiehl Here you go :)

**Comment by boris-hocde** [comment]
Thanks @sgiehl !
Sorry to bother you, could you tag & publish on NPM?

**Comment by sgiehl** [comment]
I've released a new tag/version. But I don't know if I need to do anything re npm? ping @Findus23

**Comment by Findus23** [comment]
I have published the new version to npm.
I hope it works without updating
https://github.com/matomo-org/searchengine-and-social-list/blob/master/package.json#L3

**Pull request merged**